common/src/main/java/com/hortonworks/streamline/common/exception/service/exception/request/WebserviceAuthorizationException.java

```diff
@@ -21,6 +21,6 @@
 
 public class WebserviceAuthorizationException extends WebServiceException {
     public WebserviceAuthorizationException(String msg) {
-        super(Response.Status.UNAUTHORIZED, msg);
+        super(Response.Status.FORBIDDEN, msg);
```

Comment:
@priyank5485 Why did we change this from `Response.Status.UNAUTHORIZED` to `Response.Status.FORBIDDEN`?

Reply:
@satishd I changed it because i was reading up on status codes and FORBIDDEN seems to be the right choice. For 401 Unauthorized you are supposed to have a WWW-Authenticate header which we were not passing. I am pasting a couple of links that explain the difference between the two and based on which i felt we should change it to FORBIDDEN.

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html
https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses

Reply:
@priyank5485 thanks, that makes sense to me.